### PR TITLE
RNMT-1020: Updated StatusBar to v2.4.1 + Injected viewport tag on iOS 11

### PR DESCRIPTION
**Pull request by pmaroco**

### CODE REVIEW

File: www/statusbar.js [hunk 1]
```diff
@@ -171,4 +173,15 @@ channel.deviceready.subscribe(function () {
     
   
     onVisibilityChange();
+    
+    if(device.version.split(".")[0].indexOf(IOS_11_VERSION) == 0){
```
Comment (bgracio) [inline www/statusbar.js]: Can we do this inside a function like the **onVisibilityChange**?

**Review by bgracio — CHANGES_REQUESTED** [review]

**Review by luissilvaos — CHANGES_REQUESTED** [review]
We are going to lose some important developments with this merge. I'll talk with you offline.

File: www/statusbar.js (hunk 1 above)
Comment (pmaroco) [reply www/statusbar.js]: Fixed!

**Review by bgracio — APPROVED** [review]

File: plugin.xml [hunk 2]
```diff
@@ -38,7 +38,6 @@
 
     <platform name="android">
         <source-file src="src/android/StatusBar.java" target-dir="src/org/apache/cordova/statusbar" />
-        <source-file src="src/android/ActivityAssistant.java" target-dir="src/org/apache/cordova/statusbar" />
```
Comment (luissilvaos) [inline plugin.xml]: Missing file

File: www/statusbar.js [hunk 3]
```diff
@@ -153,6 +155,21 @@ var addStatusBarDataElement = function(){
 }
 
 
+var injectViewportMetaTag = function(){
+
+    if(device.version.split(".")[0].indexOf(IOS_11_VERSION) == 0){
```
Comment (luissilvaos) [inline www/statusbar.js]: Is this the "device" object provided by the Device plugin?

**Review by luissilvaos — CHANGES_REQUESTED** [review]
Check my comments please

**Pull request closed without merging**